setup.py

```diff
@@ -102,16 +101,25 @@ def run_tests(self):
     long_description_content_type="text/markdown",
     license="BSD",
     packages=find_packages(exclude=["tests*"]),
-    install_requires=install_requires,
+    # enum34 is an enum backport for earlier versions of python
+    # funcsigs backport required for vendored debtcollector
+    # encoding using msgpack
+    install_requires=["enum34; python_version<'3.4'", "funcsigs>=1.0.0; python_version=='2.7'", "msgpack>=0.5.0",],
     extras_require={
         # users can include opentracing by having:
         # install_requires=['ddtrace[opentracing]', ...]
         "opentracing": ["opentracing>=2.0.0"],
+        "profile": ["protobuf>=3", "intervaltree",],
```

Comment:
```suggestion
        "profile": ["protobuf>=3", "intervaltree"],
```

Reply:
Do we need to document `pip install ddtrace[profile]` anywhere?

Reply:
Added a note in the doc.

Reply:
(about the suggestion, you'll have to deal with `black` ;))